.github/workflows/spotless_checks.yml

```diff
@@ -0,0 +1,17 @@
+name: Spotless Checks
+
+on:
+  schedule:
+    - cron: "0 5 * * *"
+
+jobs:
+  build:
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v1
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Spotless Apply
```

Comment:
This is exposing implementation detail that we are using Spotless to run the checks. Could we change this name to something that is tooling agnostic, like `Run code formatting checks`?